corremap.m~

```diff
@@ -0,0 +1,15 @@
+function corrmap(data,sel,unit,type)
+    % [] = RASTER_PLOT(data,sel,opt,type)
```

Comment:
Change name to CORRMAP

Reply:
Old version of the function was still lingering even though I thought it was deleted. My bad.